module.txt

```diff
@@ -1,9 +1,9 @@
 {
     "id" : "Fluid",
     "version" : "1.0.0-SNAPSHOT",
-    "author" : "Marcin Sciesinski <marcins78@gmail.com>, Mandar Juvekar",
+    "author" : "Marcin Sciesinski <marcins78@gmail.com>, Mandar Juvekar, xrtariq2594, Josharias, Cervator, flo, msteiger",
     "displayName" : "Fluid",
-    "description" : "",
+    "description" : "A module for simulating the behaviors of fluids (substances that flow). Not for liquids (natural water, lava, etc).",
```

Comment:
Remove the second sentence. Not sure what's up with the repo's description since the only fluid here is water.